frontend/src/concepts/pipelines/types.ts

```diff
@@ -68,6 +96,11 @@ export type CreatePipelineRecurringRun = (
   data: CreatePipelineRecurringRunKFData,
 ) => Promise<PipelineRecurringRunKFv2>;
 export type GetExperiment = (opts: K8sAPIOptions, experimentId: string) => Promise<ExperimentKFv2>;
+export type GetArtifact = (
+  opts: K8sAPIOptions,
+  artifactId: string,
```

Comment:
can artifactId be a number here or did that have issues

Reply:
@Gkrumbach07 , I have changed it to `number`